app/scripts/templates/sign_up.mustache

```diff
@@ -31,38 +31,27 @@
     {{/isAmoMigration}}
 
     <form novalidate>
-      <div class="input-row {{^forceEmail}}{{#showPasswordConfirm}}label-helper-top{{/showPasswordConfirm}}{{/forceEmail}}">
+      <div class="input-row {{^forceEmail}} label-helper-top{{/forceEmail}}">
         {{#forceEmail}}
           <p class="prefillEmail">{{ forceEmail }}</p>
           <input type="email" class="email hidden tooltip-below" value="{{ forceEmail }}" disabled />
         {{/forceEmail}}
         {{^forceEmail}}
-          {{#showPasswordConfirm}}
             <label class="label-helper"></label>
```

Comment:
Should we pull the indentation back to the same level as the `input`?

Reply:
👍